### PR TITLE
Fix line-height for tabs and short dsc and ratings implementaiton in product view #229 dev-37088

### PR DESCRIPTION
**Pull request by Januszpl**
Refs: #269 
dev-37088

### CODE REVIEW

File: components/04-views/product-view/_product-view.scss [hunk 1]
```diff
@@ -322,6 +328,9 @@ $product-view__wrapper-margin-bottom\@medium            : 40px !default;
         @include mq($screen-m) {
             float: none;
```
Comment (szafran89) [inline components/04-views/product-view/_product-view.scss]: Do we need this property. I know it's legacy code but please check it.

Comment (szafran89) [reply components/04-views/product-view/_product-view.scss]: Still there is `float: none;`

File: components/04-views/product-view/_product-view.scss [hunk 2]
```diff
@@ -307,7 +313,10 @@ $product-view__wrapper-margin-bottom\@medium            : 40px !default;
     &__rating {
         color: $product-view__rating-text-color;
         clear: both;
```
Comment (szafran89) [inline components/04-views/product-view/_product-view.scss]: Additionally I think we don't need ` clear: both;`

Comment (Januszpl) [reply components/04-views/product-view/_product-view.scss]: done

File: components/04-views/product-view/_product-view.scss (hunk 1 above)
Comment (Januszpl) [reply components/04-views/product-view/_product-view.scss]: done

**Pull request merged**